### ISSUE TITLE
Staging Review finding: Unclear instructions for screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93706]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** vfs-mhv-secure-messaging
**Product:** MHV Secure Messaging
**Feature:** Contact List

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

The instructions for what to do on this page --- "Select the teams you want to show..." --- aren't programmatically associated with the checkbox group. If a screen reader user tabs into the checkbox group, the instructions won't be announced to them.

This is... probably not a big deal. The purpose of this page is pretty clear and the instructions don't actually add a whole lot to the interaction. But it's possible that a user may be confused about what the intent of the interaction is and the instructions would help clarify that.

<details><summary>Link, screenshot or steps to recreate</summary>

The `va-checkbox-group` component is currently coded as:
```
<p class="vads-u-margin-bottom--0">Select the teams you want to show in your contact list when you start a new message. </p>

<form class="contactListForm">
   <div>
      <va-checkbox-group data-testid="undefined-facility-group" class="contactListFacility vads-u-margin-bottom--4 vads-u-margin-top--0 hydrated" error="" role="group">...</va-checkbox-group>
   ...
   </div>
</form>
```

![H1 heading "Contact list," followed by instruction text "Select the teams you want to show in your contact list when you start a new message" and checkboxes for the first few teams.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/90850-Brian-1.png?raw=true)

</details>

## Recommended action

This is currently coded using the [`va-checkbox-group` component](https://design.va.gov/storybook/?path=/docs/uswds-va-checkbox-group--docs), but the `label` prop isn't defined. I would recommend moving the "Select the teams you want..." instructions into the `label` prop of that component and deleting the `p` element currently containing the instructions.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by lichellebain** [comment]
https://jira.devops.va.gov/browse/MHV-62484

**Comment by bryan-riley-va** [comment]
Completed by Kyle 11/18/2024
